modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/Cache/AffinityAwarenessTest.cs

```diff
@@ -268,6 +268,32 @@ public void CacheGet_RepeatedCall_DoesNotRequestAffinityMapping()
             };
 
             Assert.AreEqual(expectedRequests, requests);
+
+            // Test cache for which affinity awareness is not applicable.
+            var cfg = new CacheClientConfiguration("replicated_cache") {CacheMode = CacheMode.Replicated};
+            var cache = Client.CreateCache<int, int>(cfg);
+
+            cache.Put(1, 1);
+            Client.CreateCache<int, int>("repeat-call-test-replicated");
+            ClearLoggers();
+
+            cache.Get(1);
+            cache.Get(1);
+            cache.Get(1);
+
+            requests = null;
```

Comment:
So we need to verify that for Replicated cache all requests go to a single (default) node, because affinity awareness does not matter. Let's state this in a comment for clarity.